tests/scripts/expect/_common.exp

```diff
@@ -64,6 +67,27 @@ proc spawn_node {id} {
     return $spawn_id
 }
 
+proc setup_leader {} {
```

Comment:
Another function `dispose_leader` should also be added here. `dispose_nodes` and other scripts calling `setup_leader` should call `dispose_leader` instead of disposing `spawn_1` directly, because the allocation and disposal should be encapsulated together.